internal/abci/accumulator.go

```diff
@@ -106,20 +106,20 @@ func (app *Accumulator) Query(reqQuery abci.RequestQuery) (resQuery abci.Respons
 		return resQuery
 	}
 
-	k, v, err := app.chain.Query(qu)
+	k, v, customErr := app.chain.Query(qu)
 	switch {
-	case err == nil:
-		// OK
+	case customErr == nil:
+		//Ok
 
-	case errors.Is(err, storage.ErrNotFound):
-		resQuery.Info = err.Error()
+	case strings.Contains(customErr.Error(), storage.ErrNotFound.Error()):
```

Comment:
With `protocol.Error.Unwrap` implemented and the return value from `queryByUrl` being passed through directly (or wrapped via `fmt.Errorf` with the `%w` format specifier), this can be reverted back to `errors.Is(err, storage.ErrNotFound)`. `strings.Contains` is not a great way to check for specific error values, given that error strings may change: https://stackoverflow.com/a/57613539/762175.